package.json

```diff
@@ -59,7 +60,9 @@
     "json-loader": "^0.5.0",
     "npm-run-all": "^3.1.0",
     "postcss": "^5.2.0",
+    "postcss-css-variables": "^0.6.0",
     "postcss-discard-duplicates": "^2.0.2",
+    "postcss-import": "^9.1.0",
```

Comment:
As @GoOz said, `cozy-bar`'s deps should not be embedded in apps but still remains in `cozy-bar` repo only. Here, the problem is that when we import `cozy-bar` in the project, webpack is looking for a `postcss.config.js` file, and it tooks the first one encoutered: the `cozy-bar`'s one.

To prevent it, we now trash the `postcss.config.js` in apps repos and put it in webpack's config file directly. See [here](https://github.com/cozy/cozy-files-v3/blob/master/config/webpack.config.base.js#L42-L46) and [there](https://github.com/cozy/cozy-files-v3/blob/master/config/webpack.config.base.js#L60-L69) (and the related [commit](https://github.com/cozy/cozy-files-v3/commit/6d25b327cbcf11eccd5a02450dc7ea75647b0d04)).

Can you do the same here? Thanks :+1: !